src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/FeedbacksController.cs

```diff
@@ -67,7 +67,9 @@ public ActionResult Create()
         /// <param name="feedbackViewModel">Feedback view model.</param>
         /// <returns>Feedback creation view.</returns>
         [HttpPost]
+#pragma warning disable S4261 // Methods should be named according to their synchronicities
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)